app/config/config.yml

```diff
@@ -32,6 +31,7 @@ framework:
     fragments:       ~
     http_method_override: true
     assets: ~
+        #version: SomeVersionScheme
```

Comment:
If the user only uncomment this line, it won't work as he also needs to remove the `~` above. Instead, I would just remove the whole comment above (`assets_version`)